packages/sw-routing/test/browser/end-to-end/regexp-route.js

```diff
@@ -0,0 +1,18 @@
+/* global goog, expect */
+
+describe('End to End Test of RegExpRoute', () => {
+  it('should work properly when there are multiple routes', () => goog.swUtils.activateSW('end-to-end/sw/regexp-route.js')
```

Comment:
Two things here:

1. Arrow functions will not work with Mocha binding - not a problem until you want to use this.timeout() or this.retries so generally I just avoid fat arrows for these callbacks to avoid future confusion.
2. Nit* I'm not a fan of arrow function straight into promise chain.